t/balancer.t

```diff
@@ -1,27 +1,35 @@
 use lib 't';
-use Test::APIcast 'no_plan';
+use Test::APIcast::Blackbox 'no_plan';
 
-$ENV{TEST_NGINX_HTTP_CONFIG} = "$Test::APIcast::path/http.d/*.conf";
-$ENV{RESOLVER} = '127.0.1.1:5353';
-
-env_to_nginx(
-    'RESOLVER'
-);
-master_on();
+repeat_each(1);
 run_tests();
 
 __DATA__
 
 === TEST 1: round robin does not leak memory
 Balancing different hosts does not leak memory.
---- http_config
-  lua_package_path "$TEST_NGINX_LUA_PATH";
-  init_by_lua_block {
-    require('resty.balancer.round_robin').cache_size = 1
+--- configuration
+  {
+      "services": [
+      {
+          "proxy": {
+            "policy_chain": [
+            { "name": "apicast.policy.upstream",
```

Comment:
instead of `upstream` poilicy, maybe better `"api_backend": "http://test:$TEST_NGINX_SERVER_PORT/",` ?

Reply:
I added it, with authentication as I did not find a straightforward way to use "api_backend" w/o authentication in other blackbox enabled tests. Is there a way to disable authentication as it does not seem to be needed for this test?

Reply:
> Is there a way to disable authentication as it does not seem to be needed for this test?

You can add a policy in the policy chain called [default_credentials](https://github.com/3scale/APIcast/blob/master/gateway/src/apicast/policy/default_credentials/apicast-policy.json). However, it is not a big deal to add `?user_key=value` as you did. I think it is ok like that

Reply:
I see, having unauthenticated requests is not a common use case. Thank you @eguzki !